src/components/button/button.js

```diff
@@ -62,7 +62,7 @@ function MdButtonDirective($mdInkRipple, $mdTheming, $mdAria) {
     if (isAnchor(attr)) {
       return '<a class="md-button" ng-transclude></a>';
     } else {
-      return '<button class="md-button" ng-transclude></button>';
+      return '<button class="md-button" type="button" ng-transclude></button>';
```

Comment:
This won't work because it won't allow users to include submit buttons. A better approach would be to provide an attribute on `<md-button>` that will pass the desired type to the transcluded button element.